routes/web.php

```diff
@@ -23,6 +23,23 @@
     return "submit successfully";
 })->name('login_accepted');
 
+Route::get('/cv', function () {
+    return view('cv');
+});
+
+#for task3 
+#the task is to send the form data to another page
+#and to show the data in a proper format
+#begin
+Route::get('/login_task3', function () {
+    return view('task3_login');
+});
+
+Route::post('/submit_Page', function () {
+    return view('task3_response', $_POST);
```

Comment:
despite it is not wrong to use super global Laravel has something called Request we will discuss this method in the session but try to search about it